src/coreComponents/linearAlgebra/DofManager.cpp

```diff
@@ -367,6 +468,12 @@ void DofManager::addField( string const & fieldName,
   addField( fieldName, location, components, support );
 }
 
+void DofManager::enableLocalReordering( string const & fieldName )
+{
+  FieldDescription & field = m_fields[getFieldIndex( fieldName )];
+  field.reorderingType = LocalReorderingType::ReverseCutHillMcKee;
```

Comment:
I quite like that you made reordering type an `enum` instead of a `bool`! I know we only have a single non-trivial reordering type, but would it make it more explicit and visible at the call site if we passed the desired `LocalReorderingType` value into this function? This way one could also "disable" it by passing a value of `None`.

Reply:
Done, this function is now called `setLocalReorderingType` and takes the reordering type as input